index.html

```diff
@@ -2450,6 +2450,46 @@ <h4>Enveloped Verifiable Credentials</h4>
 
         </section>
 
+        <section>
+          <h4>Enveloped Verifiable Presentations</h4>
+
+          <p>
+It is possible to express a [=verifiable presentation=] that has been secured
+using a securing mechanism that "envelopes" the payload, such as
```

Comment:
Is it necessary to say we use a "securing mechanism" to "secure" a VP, or can this just be a "mechanism"?
```suggestion
It is possible to express a [=verifiable presentation=] that has been secured
using a mechanism that "envelopes" the payload, such as
```